cmd/helm/create_test.go

```diff
@@ -143,8 +143,8 @@ func TestCreateStarterCmd(t *testing.T) {
 		t.Errorf("Wrong API version: %q", c.Metadata.ApiVersion)
 	}
 
-	if l := len(c.Templates); l != 6 {
-		t.Errorf("Expected 5 templates, got %d", l)
+	if l := len(c.Templates); l != 7 {
```

Comment:
Although I agree that this change keeps the spirit of the original code, might it be better to define the number of expected templates as a constant and reference to the constant+1 in the test?

Reply:
Generally speaking if it's not a core part of the PR (in this case, if it does not add a test template to `helm create`), we ask contributors to file a separate PR. It's a good suggestion, but I wouldn't say it's something that needs to be done to accomplish what this PR is trying to solve.

However, if you feel like submitting a new PR to improve the tests, that would be much appreciated @henrynash.

Reply:
Agree entirely with this approach. I'll submit a follow-on PR.

Reply:
Ok, thanks for feedback @bacongobbler. @henrynash going to leave it as is, as its not related to the PR and not an issue either.